spec/support/shared/with_completed_course.rb

```diff
@@ -0,0 +1,72 @@
+RSpec.shared_context 'with completed course' do
```

Comment:
This is very similar to an existing shared context is it necessary. It is a whole lot of set up just to use in a single assertion?